### ISSUE TITLE
Build failure: `_7zz`

### ISSUE THREAD

**Issue by griffi-gh** [NixOS/nixpkgs#353493]
### Steps To Reproduce

1. build `_7zz`

### Build log

```
error: builder for '/nix/store/qqp9cdcplmnxkg1jm8xlvpsnz2kdqxhb-7zz-24.08.drv' failed with exit code 2;
       last 25 log lines:vU%3D' (2024-10-29)
       > patching file C/Util/7zipInstall/resource.rc.rej
       > patching file C/Util/7zipUninstall/resource.rc
       > patching file C/Util/7zipUninstall/resource.rc.rej
       > patching file CPP/7zip/7zip_gcc.mak
       > patching file CPP/7zip/Crypto/RandGen.cpp
       > patching file CPP/7zip/GuiCommon.rc
       > patching file CPP/7zip/UI/FileManager/PanelItemOpen.cpp
       > patching file CPP/7zip/UI/FileManager/SysIconUtils.h
       > patching file CPP/Windows/Control/ComboBox.h
       > patching file CPP/Windows/Control/ImageList.h
       > patching file CPP/Windows/Control/ListView.h
       > patching file CPP/Windows/Control/ProgressBar.h
       > patching file CPP/Windows/SecurityUtils.h
       > Running phase: updateAutotoolsGnuConfigScriptsPhase
       > Running phase: configurePhase
       > no configure script, doing nothing
       > Running phase: buildPhase
       > build flags: -j12 SHELL=/nix/store/mm0pa3z7kk6jh1i9rkxqxjqmd8h1qpxf-bash-5.2p37/bin/bash CC=cc CXX=c++ DISABLE_RAR_COMPRESS=true
       > mkdir -p b/g_x64
       > asmc -nologo -elf64 -DABI_LINUX -Fob/g_x64/ ../../../../Asm/x86/LzmaDecOpt.asm
       > cc      -O2 -c -Werror -Wall -Wextra -Waddress -Waggressive-loop-optimizations -Wattributes -Wcast-align -Wcomment -Wdiv-by-zero -Wformat-contains-nul -Winit-self -Wint-to-pointer-cast -Wunused -Wunused-macros  -Wbool-compare -Wduplicated-cond  -Waddress-of-packed-member -Wbool-operation -Wcast-align=strict -Wconversion -Wdangling-else -Wduplicated-branches -Wimplicit-fallthrough=5 -Wint-in-bool-context -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-attributes -DNDEBUG -D_REENTRANT -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -fPIC   -o b/g_x64/7zBuf2.o ../../../../C/7zBuf2.c
       > cc      -O2 -c -Werror -Wall -Wextra -Waddress -Waggressive-loop-optimizations -Wattributes -Wcast-align -Wcomment -Wdiv-by-zero -Wformat-contains-nul -Winit-self -Wint-to-pointer-cast -Wunused -Wunused-macros  -Wbool-compare -Wduplicated-cond  -Waddress-of-packed-member -Wbool-operation -Wcast-align=strict -Wconversion -Wdangling-else -Wduplicated-branches -Wimplicit-fallthrough=5 -Wint-in-bool-context -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-attributes -DNDEBUG -D_REENTRANT -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -fPIC   -o b/g_x64/7zCrc.o ../../../../C/7zCrc.c
       > /nix/store/mm0pa3z7kk6jh1i9rkxqxjqmd8h1qpxf-bash-5.2p37/bin/bash: line 1: asmc: command not found
       > make: *** [../../7zip_gcc.mak:1310: b/g_x64/LzmaDecOpt.o] Error 127
       > make: *** Waiting for unfinished jobs....
       For full logs, run 'nix log /nix/store/qqp9cdcplmnxkg1jm8xlvpsnz2kdqxhb-7zz-24.08.drv'.

```

### Additional context

N/A

### Notify maintainers

@anna328p @peterhoeg 

### Metadata

---

- system: `"x86_64-linux"`
 - host os: `Linux 6.11.5-zen1, NixOS, 24.11 (Vicuna), 24.11.20241029.807e915`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.25.0pre20240920_ca3fc169`
 - nixpkgs: `/nix/store/m68ikm8045fj7ys7qvgr608z9l70hh1k-source`
 
Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by FliegendeWurst** [comment]
Probably a duplicate of #353119

**Comment by itepastra** [comment]
can you check if its fixed by #353272?

**Comment by JohnRTitor** [comment]
Will be fixed once the channels update, thanks for the report!

**Comment by JohnRTitor** [comment]
Fixed by #353272
